src/sass/_theme_layout.sass

```
@@ -45,18 +45,13 @@ html
   ul
     margin-bottom: 0
   li
-    &.divide-top
-      border-top: solid 1px hsl(0, 0%, 25%)
-    &.divide-bottom
-      border-bottom: solid 1px hsl(0, 0%, 25%)
     &.current
       background: darken($menu-vertical-background-color, 10%)
       a
         color: $menu-link-medium
-        border-right: solid 1px darken($menu-vertical-background-color, 20%)
         padding: ($gutter / 4) ($gutter * 1.5)
         &:hover
-          background: darken($menu-vertical-background-color, 15%)
+          background: $menu-vertical-background-color
```

```suggestion
          box-shadow: inset 0 0 100px #eaeaeaaa
```
Changing this from background to box-shadow would allow the `:hover` color to overlay each level sub-menu background color resulting in a smoother effect.